pkg/services/cloudmigration/cloudmigrationimpl/snapshot_mgmt.go

```diff
@@ -289,11 +296,17 @@ func (s *Service) buildSnapshot(ctx context.Context, signedInUser *user.SignedIn
 			Data:  item.Data,
 		})
 
+		parentName := ""
+		if _, exists := migrationData.ItemParentNames[item.Type]; exists {
+			parentName = migrationData.ItemParentNames[item.Type][item.RefID]
+		}
```

Comment:
We can simplify this because:

```go
var m map[string]string
fmt.Println(m["key-that-doesnt-exist"] == "") // true
```

```suggestion
		parentName := migrationData.ItemParentNames[item.Type]
```

Reply:
That has another level of access `[item.RefID]` though, it's a map of map string

Reply:
Oh yeah, missed that